corehq/apps/reports/templates/reports/reportdata/case_details.html

```diff
@@ -250,6 +250,7 @@ <h4 class="modal-title">
     {% endif %}
 
     {% if can_edit_data %}
+    {% if show_properties_edit or show_case_rebuild or not case.closed and not is_usercase %}
```

Comment:
Yeah, I despise this pattern, it's all over app manager (complicated boolean to determine whether or not to show a section). Django templates don't support parens here (to prevent expressions in templates from getting too complicated), and naming it as a var doesn't work for the same reason. Added a comment.